### ISSUE TITLE
test_version_update_at_roles_reload fails on 1.10 and 2.7

### ISSUE THREAD

**Issue by Totktonada** [tarantool/cartridge-cli#650]
1.10 and 2.7 (but not 2.8) fail on `test_version_update_at_roles_reload`:

```
=========================== short test summary info ============================
FAILED test/e2e/test_tgz.py::test_version_update_at_roles_reload - docker.err...
ERROR test/e2e/test_tgz.py::test_tgz - AssertionError: Error during creating ...
=============== 1 failed, 4 passed, 1 error in 489.08s (0:08:09) ===============
```

**Comment by DifferentialOrange** [comment]
Example: https://github.com/tarantool/cartridge-cli/runs/4325924032?check_suite_focus=true

**Comment by DifferentialOrange** [comment]
https://github.com/tarantool/cartridge-cli/runs/4521568155?check_suite_focus=true

**Comment by DifferentialOrange** [comment]
It looks like https://github.com/tarantool/cartridge-cli/runs/4325924032?check_suite_focus=true issues were caused by change in installer politics https://github.com/tarantool/installer.sh/commit/841bd7a9de43adf7663fc3674a94889b2110b91f and it was fixed in #629

**Comment by DifferentialOrange** [comment]
Fails do not reproduce both locally and on GitHub runners

**Comment by DifferentialOrange** [comment]
https://github.com/tarantool/cartridge-cli/runs/4521568155?check_suite_focus=true case looks like temporary issues with Tarantool servers (see logs), issue do not reproduces.

**Comment by DifferentialOrange** [comment]
@Totktonada , I propose this ticket to be "closed by #629"

**Comment by Totktonada** [comment]
Thanks for the investigation! Okay, closing as fixed in PR #629.
